src/systems/ec/implementations/react/components/page-header-core/src/PageHeaderCore.jsx

```diff
@@ -25,7 +25,9 @@ const PageHeaderCore = ({
         {React.cloneElement(breadcrumb, {
           className: 'ecl-page-header-core__breadcrumb',
         })}
-        {meta && <div className="ecl-page-header-core__meta">{meta}</div>}
+        {meta && (
+          <div className="ecl-page-header-core__meta">{meta.toUpperCase()}</div>
```

Comment:
@planctus in the ticket it says the uppercase change for the meta should happen via css, but you are setting it on the template. Is this intended ?